specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_CreateOrUpdate.json

```diff
@@ -0,0 +1,156 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "amlFilesystemName": "fs1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview",
+    "amlFilesystem": {
+      "tags": {
+        "Dept": "ContosoAds"
+      },
+      "location": "westus",
+      "properties": {
+        "storageCapacityTiB": 3072,
+        "zones" : ["1", "2"],
+        "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/scvnet/subnets/sub1",
+        "MSGAddress" : "10.0.0.4",
+        "mountCommand" : "mount -t lustre 10.1.0.13@tcp:/lustrefs /lustre/lustrefs",
+        "lustreVersion" : "2.12.5",
+        "encryptionSettings": {
+          "keyEncryptionKey": {
+            "keyUrl": "https://examplekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+            "sourceVault": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/examplekv"
+          }
+        },
+        "maintenanceWindow" : {
+          "dayOfWeek": "Friday",
+          "timeOfDay" : "22:00"
+        },
+        "HSMSettings" : {
+          "container" : "https://somecontainer.blob.core.windows.net/containername",
+          "containerKeyVaultKey" : {
+            "keyUrl" : "https://somekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+            "sourceVault" : "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/somekv"
+          },
+          "importPrefix" : "asc",
+          "exportPrefix" : "asc"
+        }
+      },
+      "sku": {
+        "name": "Standard_2G"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "tags": {
+          "Dept": "ContosoAds"
+        },
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/amlFilesystems/fs1",
+        "location": "westus",
+        "name": "fs1",
+        "type": "Microsoft.StorageCache/amlFilesystem",
+        "properties": {
+          "storageCapacityTiB": 3072,
+          "zones" : ["1", "2"],
+          "health": {
+            "state": "Healthy",
+            "statusDescription": "amlFilesystem is ok."
+          },
+          "provisioningState": "Succeeded",
+          "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/scvnet/subnets/sub1",
+          "MSGAddress" : "10.0.0.4",
+          "mountCommand" : "mount -t lustre 10.1.0.13@tcp:/lustrefs /lustre/lustrefs",
```

Comment:
In the common case, the address in mountCommand will be the same as MGSAddress

Reply:
Updated in next commit.